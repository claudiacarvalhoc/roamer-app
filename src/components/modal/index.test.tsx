import React from 'react';
import Modal from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const getShallowWrapper = () => {
    return shallow(
    <Modal>
        <p>{'Welcome!'}</p>
    </Modal>);
};

describe('<Modal />', () => {
    it('render correctly', () => {
        const component = getShallowWrapper();
        expect(toJson(component)).toMatchSnapshot();
    });
});
