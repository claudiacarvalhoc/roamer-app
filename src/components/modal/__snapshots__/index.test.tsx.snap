// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> render correctly 1`] = `
<div
  className="modal"
>
  <div
    className="content"
  >
    <p>
      Welcome!
    </p>
  </div>
</div>
`;
